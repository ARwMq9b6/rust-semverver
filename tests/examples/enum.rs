pub mod old {
    pub enum Abc {

    }

    pub enum Bcd {
        Abc,
    }

    pub enum Cde {
        Abc,
    }

    pub enum Def {
        Abc,
        Bcd,
    }
}

pub mod new {
    pub enum Abc {
        Abc,
    }

    pub enum Bcd {

    }

    pub enum Cde {
        Abc,
        Bcd,
    }

    pub enum Def {
        Abc,
    }
}
