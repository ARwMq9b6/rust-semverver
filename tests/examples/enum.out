version bump: 1.0.0 -> (Breaking) -> 2.0.0
warning: Breaking changes in `Abc`
  --> $REPO_PATH/tests/examples/enum.rs:21:5
   |
21 | /     pub enum Abc {
22 | |         Abc,
23 | |     }
   | |_____^
   |
warning: Breaking (VariantAdded)
  --> $REPO_PATH/tests/examples/enum.rs:22:9
   |
22 |         Abc,
   |         ^^^

warning: Breaking changes in `Bcd`
  --> $REPO_PATH/tests/examples/enum.rs:25:5
   |
25 | /     pub enum Bcd {
26 | |
27 | |     }
   | |_____^
   |
warning: Breaking (VariantRemoved)
  --> $REPO_PATH/tests/examples/enum.rs:7:9
   |
7  |         Abc,
   |         ^^^

warning: Breaking changes in `Cde`
  --> $REPO_PATH/tests/examples/enum.rs:29:5
   |
29 | /     pub enum Cde {
30 | |         Abc,
31 | |         Bcd,
32 | |     }
   | |_____^
   |
warning: Breaking (VariantAdded)
  --> $REPO_PATH/tests/examples/enum.rs:31:9
   |
31 |         Bcd,
   |         ^^^

warning: Breaking changes in `Def`
  --> $REPO_PATH/tests/examples/enum.rs:34:5
   |
34 | /     pub enum Def {
35 | |         Abc,
36 | |     }
   | |_____^
   |
warning: Breaking (VariantRemoved)
  --> $REPO_PATH/tests/examples/enum.rs:16:9
   |
16 |         Bcd,
   |         ^^^

